privacy-snapshot.json

```diff
@@ -12,6 +12,7 @@
   "bafkreifvhjdf6ve4jfv6qytqtux5nd4nwnelioeiqx5x2ez5yrgrzk7ypi.ipfs.dweb.link",
   "bafybeidxfmwycgzcp4v2togflpqh2gnibuexjy4m4qqwxp7nh3jx5zlh4y.ipfs.dweb.link",
   "bridge.api.cx.metamask.io",
+  "bridge.dev-api.cx.metamask.io",
```

Comment:
Was this added to fix unit tests locally or in CI? This shouldn't be necessary in CI

Locally I run the tests with the BRIDGE_USE_DEV_APIS env var commented out and to avoid the privacy snapshot failure

Reply:
Pushed up a commit to check if it fails in CI

Reply:
CI did not break so we can keep this commit